batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/StudentDao.java

```diff
@@ -0,0 +1,175 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.utils.DbConnection;
+
+import javax.naming.NamingException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @Author binodnme
+ * Created on 1/14/16
+ */
+public class StudentDao {
+
+    public List<Student> fetch(Integer page) throws DataException {
+        List<Student> studentList = new ArrayList<Student>();
+        try {
+            Connection conn = DbConnection.getConnection();
+
+            Integer totalDataToFetch = 30;
+            Integer offset = (page-1)*totalDataToFetch;
+            String query = "select * from student limit "+totalDataToFetch+" Offset "+offset;
+//            String query = "select * from student ";
+//            String query = "select * from student limit ? offset ?";
+//            PreparedStatement ps = conn.prepareStatement(query);
+//            ps.setInt(1,totalDataToFetch);
+//            ps.setInt(2,offset);
+            Statement stmt = conn.createStatement();
+            ResultSet studentResult = stmt.executeQuery(query);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2077)